lib/pf/version.pm

```diff
@@ -0,0 +1,120 @@
+package pf::version;
+=head1 NAME
+
+pf::version add documentation
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::version
+
+=cut
+
+use strict;
+use warnings;
+
+use pf::constants;
+use pf::file_paths;
+use pf::db;
+use pf::log;
+
+use constant PF_VERSION => 'version';
+# The next two variables and the _prepare sub are required for database handling magic (see pf::db)
+our $version_db_prepared = 0;
+
+# In this hash reference we hold the database statements. We pass it to the query handler and it will repopulate
+# the hash if required.
+our $version_statements = {};
+
+=head1 SUBROUTINES
+
+=head2 version_db_prepare
+
+Initialize database prepared statements
+
+=cut
+
+sub version_db_prepare {
+
+    $version_statements->{'version_check_sql'} = get_db_handle()->prepare(qq[
+            SELECT version FROM pf_version WHERE version = ?;
+    ]);
+    $version_db_prepared = 1;
+    return 1;
+}
+
+=head2 version_check
+
+Checks the version of db
+
+=cut
+
+sub version_check {
```

Comment:
We are in pf::version class. I'm just wondering if we could simply rename version_check to check... ?
Something like what I've done in pf::iplog...